app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,9 @@
   </head>
 
   <body>
+    <nav class="navbar">
+    <%= link_to image_tag("/logo.png", border: 0), {action: 'home', controller: '/pages'}, 
```

Comment:
Should work without forwarding slash.
Also, use route helper instead of `{action: 'home', controller: '/pages'}`.
And make this link_to in one line